### DISCUSSION TITLE
Target list gone

### DISCUSSION THREAD

**Discussion by Czasemsprytny** [DinoDevs/GladiatusCrazyAddon#483] [Q&A] [answered]
Right after the creza adon update, my goal list was reset. Why? I had a lot of data there, is it all lost?My list had 300 players, what about it?

**Comment by FrutyX** [comment, marked as answer]
Hey, sorry for the late response.

You mean the Target list for players? Hmm, that might have been a side effect of GCA update, the target list is stored within your browser by GCA.

However, there is a way to prevent this, especially if you have such long list, in GCA settings - Data section, you can export your list (settings) into a file (or into gladiatus notes) and then import it back, meaning you will always a backup for situations like this.

Sorry for the inconvenience.
